benchmarks/synchrobench/src/main/java/com/oath/oak/synchrobench/maps/OakMap.java

```diff
@@ -13,10 +13,10 @@
     private com.oath.oak.OakMap<MyBuffer, MyBuffer> oak;
     private OakMapBuilder<MyBuffer, MyBuffer> builder;
     private MyBuffer minKey;
-    private OakNativeMemoryAllocator ma;
+    public OakNativeMemoryAllocator ma;
```

Comment:
What is the need for this change?

Reply:
In order to print the memory allocated there needs to be access to the memory allocator.
I guess a getter can be added instead if necessary.